SampleApp/Project.swift

```diff
@@ -12,7 +12,14 @@ let project = Project(
 
         .app(
             name: "SampleApp",
-            sources: ["Sources/**"]
+            sources: ["Sources/**"],
+            scripts: [
+                .pre(
+                    path: .relativeToRoot("../Scripts/extract_english_strings.sh"),
```

Comment:
Hm, not a blocker, but I kind of think this should be a fully manual step since we are not expecting additional localizations on the regular.